examples/build-all.sh

```diff
@@ -117,6 +117,20 @@ function webpack2 {
   )
 }
 
+function webpack2ExtractText {
+  echo "Testing HappyPack with webpack v2 + extract-text-plugins + react"
+  echo "---------------------------------"
+
+  setup_example "examples/webpack2-extracText-react"
+
+  (
+    cd examples/webpack2-extracText-react;
```

Comment:
not a big deal, but there's a typo in the example name where it should've been something like webpack2-extract-text instead of extracText (note the missing T)